csun-metalab/nomi
# NAM-117 -- ENG: Change mark as recognized

# Description 

Removed the old recognition functionality and updated it to be directly associated with what side of a flash card is currently being shown. Also removed the ability to shuffle photo gallery cards

# Installation

n/a

# Testing instructions

Check to make sure photo gallery cards cannot be shuffled
Then, after flipping a few flash cards, make sure that all face up cards are moved and shuffled to the top, and all name up cards are moved and shuffled to the bottom
